elizhahn/karaoke-stretch
# Testing/home mysongs and songbook

## What does this PR do (summary of changes)?
 - This updates the testing suites for the current functionality
 - Fixtures were added to mock the data coming from our API and intercepts were set up for test efficiency
 - testing attributes were added to the elements for targeting
## How should it be tested?
 - Run the testing suite and make sure they are all passing, 
 - Am I missing any user flows currently that I could add to this branch?
## Future Iterations:
 - Will need to add testing for error messages we display to the user once built out
